cmd/ci-secret-generator/main.go

```diff
@@ -120,17 +133,54 @@ func (o *options) validateConfig() error {
 }
 
 func executeCommand(command string) ([]byte, error) {
-	out, err := exec.Command("bash", "-o", "errexit", "-o", "nounset", "-o", "pipefail", "-c", command).CombinedOutput()
-	if err != nil {
-		return nil, fmt.Errorf("%s : %w", string(out), err)
+	cmd := exec.Command("bash", "-o", "errexit", "-o", "nounset", "-o", "pipefail", "-c", command)
+	var outBuf, errBuf bytes.Buffer
+	cmd.Stdout = &outBuf
+	cmd.Stderr = &errBuf
+
+	if err := cmd.Run(); err != nil {
+		stderr := errBuf.Bytes()
+		stdout := outBuf.Bytes()
+		partialStreams := true
+		// The command completed with non zero exit code, standard streams *should* be available.
+		if _, ok := err.(*exec.ExitError); ok {
+			partialStreams = false
+		}
```

Comment:
```suggestion
		// The command completed with non zero exit code, standard streams *should* be available.
		_, partialStreams := err.(*exec.ExitError)
```

Reply:
I feel `partialStreams` is negated comparing to the original logic.

Reply:
I'm not sure about that: if the error is `*exec.ExitError` then streams should be ok, and that means `partialStreams = false`

Reply:
Yes, sorry, edited too quickly.  But you get the point, there's no need for the conditional.